package.json

```diff
@@ -4,13 +4,13 @@
   "private": true,
   "homepage": "https://codingcoach.io/",
   "scripts": {
-    "start": "yarn build:css && node scripts/custom-config start",
-    "build": "yarn build:css && node scripts/custom-config build",
+    "start": "yarn build:css && react-scripts start",
```

Comment:
I was looking at `create reac app` and it only had `react-scripts start`. Can we not use `yarn build:css`?

Reply:
If we don't do this, then we will have to build the styles manually before starting the project (which sometimes we might forget), given that we don't have full control over our build process.

I'd definitely like to improve this, but for now I think is ok. Let's create an issue to improve it.